### DISCUSSION TITLE
Quick search not working after upgrade ... was working on v3.0.11

### DISCUSSION THREAD

**Discussion by AlexIT-FT** [netbox-community/netbox#9552] [General]
### NetBox version

v3.2.4

### Python version

3.8

### Steps to Reproduce

Coming from v.3.0.11, just upgraded to v3.2.4

Open any list (sites, regions, devices, ..)
Type anything in quick search box, list not changes.

### Expected Behavior

Live filtering the list , like in previous version

### Observed Behavior

List not changes.
No error in debug console.

Generic CORS error always on console, but I don't think it's relevant
The Cross-Origin-Opener-Policy header has been ignored, because the URL's origin was untrustworthy. It was defined either in the final response or a redirect. Please deliver the response using the HTTPS protocol. You can also use the 'localhost' origin instead. See https://www.w3.org/TR/powerful-features/#potentially-trustworthy-origin and https://html.spec.whatwg.org/#the-cross-origin-opener-policy-header.

**Comment by kkthxbye-code** [comment]
I'll convert this to a discussion as it is more of a support request than a bug.

**Reply by AlexIT-FT** [reply]
Sorry but , maybe I'm missing something.
If "quick search" was working in v3.0.11 and not working in v3.2.4, Why isn't it a bug?

**Reply by jeremystretch** [reply]
This is not a bug because you can clearly see that quick search _is_ working e.g. on the [public demo](https://demo.netbox.dev), and you did not provide steps to reproduce the issue. Therefore it is highly likely that it is an issue with your specific deployment.

**Comment by AlexIT-FT** [comment]
You're right, and I found it!
I 've extract new version over existing one, some files of older version was in conflig with new ones.
I've upgraded again in a separate folder, copying just needed files from running instance, and quick-search now is working as usual.
Thanks and sorry for wasting your time.
